classes/form/section_select_form.php

```diff
@@ -45,6 +45,8 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 class section_select_form extends moodleform {
+    /** Int array of allowed values */
+    protected $allowedvalues = [];
```

Comment:
the variable name does not really mean the values.
maybe $targetsections or something is better?

Reply:
Hi @TomoTsuyuki 

Thanks for your review, I have changed it to $allowedsections, which only contains section numbers which are allowed to be chosen.